rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_accessor :date, :student, :book
+
+  def initialize(date)
+    @date = date
+    @person = person
+    person.add_rental(self)
+    @book = book
+    book.add_rental(self)
```

Comment:
- [ ] In achieving "A rental belongs to a book and also belongs to a person", you need to fix a  couple of changes here:
- The constructor of `Rental` should be modified so `Book and Person` are set in it to break the `many-to-many `relationship. Link to the example article: https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship
  
  ### Sample code:
```
def initialize(date, book, person)

    @book = book
    # The rental is adding itself to the @book achieving the 'belongs-to' association
    book.rentals << self

    @person = person
    # The rental is adding itself to the @person the 'belongs-to' association
    person.rentals << self
  end
```

Reply:
well resolved! ✔️